tests/legacy-cli/e2e/utils/assets.ts

```diff
@@ -51,10 +51,6 @@ export async function createProjectFromAsset(
   process.chdir(dir);
   if (!useNpmPackages) {
     await useBuiltPackages();
-    if (!getGlobalVariable('ci')) {
```

Comment:
fwiw the registry is controlled using the environment variables. I think those variables even override this file so this was probably never doing anything?